### PR TITLE
Update delete resource modal texts

### PR DESCRIPTION
**Pull request by rohitkrai03**
## Fixes 

- https://issues.redhat.com/browse/HAC-2365
- https://issues.redhat.com/browse/HAC-2364
- https://issues.redhat.com/browse/HAC-2363
- https://issues.redhat.com/browse/HAC-2358
- https://issues.redhat.com/browse/HAC-2359
- https://issues.redhat.com/browse/HAC-2360

## Description

- Updated delete application modal texts based on UX recommendation.
- Updated delete component modal texts based on UX recommendation.

## Type of change

- [x] Bugfix

## Screen shots / Gifs for design review 

https://user-images.githubusercontent.com/6041994/209315998-cbe2aa92-dc5f-45a8-8c65-32e635975a03.mov

## Browser conformance: 

- [x] Chrome
- [ ] Firefox
- [ ] Safari
- [ ] Edge

### CODE REVIEW

File: src/components/modal/resource-modals.tsx [hunk 1]
```diff
@@ -8,6 +8,13 @@ export const applicationDeleteModal = (applicationObj: ApplicationKind) =>
     obj: applicationObj,
     model: ApplicationModel,
     displayName: applicationObj.spec.displayName,
+    description: (
+      <>
+        The application <strong>{applicationObj.spec.displayName}</strong> will be deleted
+        permanently with all of it’s components. The deleted application will be promoted manually
```
Comment (christianvogt) [inline src/components/modal/resource-modals.tsx:14]:
```suggestion
        permanently with all of its components. The deleted application will be promoted manually
```

Comment (rohitkrai03) [reply src/components/modal/resource-modals.tsx]: Updated.

**Pull request merged**